exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,7 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  return (arr = arr.length === 0); // complete this statement
```

Comment:
Good job checking if `arr.length` is equal to zero. We don't need the variable assignment here however (`arr = `).